config/locales/views/reactions/en.yml

```diff
@@ -33,6 +33,3 @@ en:
         aria_label: Add to reading list
         icon: Save button
         text: Save
-      readinglist:
```

Comment:
This key _seems_ redundant -- the views consistently call `readingList`. With everything currently in-flight with `multiple_reactions`, this change makes me a _bit_ nervous, but I've been testing it locally and as far as I can tell, it appears that everything works as expected. At the same time, given that we're attempting to become [_more_ consistent with our reaction emojis](https://github.com/forem/forem/pull/19116), I am wondering if we'd rather shift the views to `readinglist`, rather than removing the redundant key. It could be that this is a temporary change, but that seems okay to me, all things considered?

Reply:
Thank you for the information. I did wonder if this is for future expansion or a debug remnant, but eventually decided to be the latter because the French and English keys don't match up and nobody is calling them. Name change sounds a legitimate reason, but in that case we probably have to have the same number of keys you need in all languages (lest it crash UI in either locale). If you are not sure how many you need, then probably not yet ready to add...